views/js/runner/plugins/controls/timer/timers.js

```diff
@@ -85,9 +85,10 @@ define([
      * @param {Object} [config] - timers config
      * @param {Object[]} [config.warnings] - the warnings to apply to the timers (max only for now)
      * @param {Object[]} [config.warnings] - the warnings to apply to the timers (max only for now)
+     * @param {Object[]} extraTime - as defined in the testContext
```

Comment:
Please update the documented type accordingly `{Object]` instead of `{Object[]}`